db/migrations/00005_create_storage_diffs_table.sql

```diff
@@ -8,6 +8,7 @@ CREATE TABLE public.storage_diff
     storage_key    BYTEA,
     storage_value  BYTEA,
     checked        BOOLEAN NOT NULL DEFAULT FALSE,
+    from_backfill  BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
I'd be happy to remove this change until after the release if we don't want to change the schema this last minute.

Reply:
Would be up for merging the migrations early/separately too if you think merging this PR will happen after the release